index.html

```diff
@@ -69,7 +69,7 @@
 
                 <div class="outside-in-headerTitle">
                         <span class="outside-in-headerTitle-icon"></span>
-                        <h1>OUTSIDE·IN</h1>
+                        <h1>OUTSIDE•IN</h1>
```

Comment:
@dayotte, this bullet point (and the one at the bottom) isn't rendering in the browsers – it just looks like an empty space. You'll have to try to revert to the character that was there before. Try copying the old character from the 'diff' view in github, that seems to work.